R/data.R

```diff
@@ -1852,7 +1873,7 @@
 #'   \item{pres_or_congress}{President or congress that issued action}
 #'   \item{acres_affected}{Acres affected by action. Note that total current acreage is not included. National monuments that cover ocean are listed in square miles.}
 #' }
-#' @source National Parks Conservation Association \url{https://www.npca.org/} and National Parks Service Archeology Program \url{https://www.nps.gov/history/archeology/sites/antiquities/MonumentsList.htm}
+#' @source National Parks Conservation Association \url{https://www.npca.org/} and National Parks Service Archeology Program https://www.nps.gov/history/archeology/sites/antiquities/MonumentsList.htm
```

Comment:
Here too

Reply:
same

Reply:
OK. I didn't know how finicky URLs were for CRAN so I figured we could just remove them all, but if it works to only remove a few that's fine by me.